src/DynamoCore/Graph/Nodes/CustomNodes/Function.cs

```diff
@@ -514,24 +520,27 @@ protected override void DeserializeCore(XmlElement nodeElement, SaveContext cont
 
             ArgumentLacing = LacingStrategy.Disabled;
         }
+        
 
         private bool TryParseInputExpression(string inputSymbol, 
                                              out IdentifierNode identifier, 
                                              out AssociativeNode defaultValue,
-                                             out string comment)
+                                             out string comment,
+                                             out List<ErrorEntry> errors)
```

Comment:
it's good for api's to use a general interface if possible - IEnumerable<ErrorEntry> is more appropriate unless you really need a list.